### ISSUE TITLE
Next.js Icon

### ISSUE THREAD

**Issue by silham** [Remix-Design/RemixIcon#673]
## Icon Request

* Icon name: Next.js
* Tags of this icon: nextjs, dev, js
* Use case: web deelopment
* Screenshots of similar icons:
![image](https://github.com/Remix-Design/RemixIcon/assets/79428366/aa482a59-67dd-43af-a6e6-c1e24c4ab8fd)

**Comment by xiaochunjimmy** [comment]
Remixicon does not support gradients, so we cannot add this icon.

**Comment by nico-frey** [comment]
> Remixicon does not support gradients, so we cannot add this icon.

Good point; Next JS has a non-gradient variant if you still want to consider adding it. I personally don't need it though.

![image](https://github.com/Remix-Design/RemixIcon/assets/49398439/f97de9dc-d79c-426d-9dcf-cf29c7d4f713)

**Comment by silham** [comment]
Yeah, that's fine. Can you consider adding this?

**Comment by xiaochunjimmy** [comment]
https://remixicon.com/icon/nextjs-line
https://remixicon.com/icon/nextjs-fill
